File: Topic Based Problems/01 Fundamentals/1 Beginner/16 Swap Two Numbers/Python/SwapTwoNumbers.py
firstNumber = int(input("Enter first number: "))
secondNumber = int(input("Enter second number: "))

print("Before swapping:")
print("First number:", firstNumber)
print("Second number:", secondNumber)

firstNumber = firstNumber + secondNumber
secondNumber = firstNumber - secondNumber
firstNumber = firstNumber - secondNumber

print("After swapping:")
print("First number:", firstNumber)
print("Second number:", secondNumber)

File: Topic Based Problems/01 Fundamentals/1 Beginner/16 Swap Two Numbers/Java/SwapTwoNumbers.java
import java.util.Scanner;

public class SwapTwoNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int firstNumber = scanner.nextInt();

        System.out.print("Enter second number: ");
        int secondNumber = scanner.nextInt();

        System.out.println("Before swapping: ");
        System.out.println("First number: " + firstNumber);
        System.out.println("Second number: " + secondNumber);

        firstNumber = firstNumber + secondNumber;
        secondNumber = firstNumber - secondNumber;
        firstNumber = firstNumber - secondNumber;

        System.out.println("After swapping: ");
        System.out.println("First number: " + firstNumber);
        System.out.println("Second number: " + secondNumber);

        scanner.close();
    }
}
